### ISSUE TITLE
Uncaught TypeError: Cannot use 'in' operator to search for 'executablePath' in null

### ISSUE THREAD

**Issue by NuclearCookie** [gogoprog/atom-gdb#7]
Tested on Linux Mint 17.2: 
1. Installed atom gdb
2. pressed f5

**Atom Version**: 1.0.5
**System**: Linux Mint
**Thrown From**: [atom-gdb](https://github.com/gogoprog/atom-gdb) package, v0.2.0
### Stack Trace

Uncaught TypeError: Cannot use 'in' operator to search for 'executablePath' in null

```
At /home/pieter/.atom/packages/atom-gdb/lib/atom-gdb.coffee:45

TypeError: Cannot use 'in' operator to search for 'executablePath' in null
  at Object.module.exports.AtomGdb.getSetting (/home/pieter/.atom/packages/atom-gdb/lib/atom-gdb.coffee:45:43)
  at Object.module.exports.AtomGdb.start (/home/pieter/.atom/packages/atom-gdb/lib/atom-gdb.coffee:85:12)
  at atom-workspace.subscriptions.add.atom.commands.add.atom-gdb:start (/home/pieter/.atom/packages/atom-gdb/lib/atom-gdb.coffee:27:82)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/usr/share/atom/resources/app.asar/src/command-registry.js:241:29)
  at /usr/share/atom/resources/app.asar/src/command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/usr/share/atom/resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:524:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/usr/share/atom/resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:347:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (/usr/share/atom/resources/app.asar/src/window-event-handler.js:177:20)

```
### Commands

```
     -0:39.2.0 editor:delete-to-beginning-of-word (atom-text-editor.editor.mini.is-focused)
     -0:38.4.0 core:backspace (atom-text-editor.editor.mini.is-focused)
     -0:31 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:00.7.0 atom-gdb:start (atom-text-editor.editor.is-focused)
```
### Config

``` json
{
  "core": {
    "themes": [
      "one-dark-ui",
      "solarized-dark-syntax"
    ]
  },
  "atom-gdb": {}
}
```
### Installed Packages

``` coffee
# User
atom-gdb, v0.2.0

# Dev
No dev packages
```

**Comment by gogoprog** [comment]
Thanks for reporting, it must be related to a fresh install, I'll fix it asap

**Comment by gogoprog** [comment]
Fix in latest release 0.2.1
